### ISSUE TITLE
BE | Maintenance Period Alerts | Outline and review development steps

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#89280]
## Background 
Maintenance window creation in pager duty is currently manual. We think we could create a utility that:

- Monitors downtime and maintenance notifications from Lighthouse.
- Provides an administrative interface for managing maintenance windows.
- Automatically creates and publishes maintenance windows.
- Sends notifications about maintenance window creation to Slack.

See discovery ticket:
 - #89282 

## Acceptance Criteria
- [ ] ~we've got working tool for automating maint. windows (proof of concept app - something we can share out w/others)~ 
- [x] we've broken the work down to automate these windows.  

### Tasks/Tickets we created for this work. See the epic #92169
- [ ] Polling for maintenance notifications
     - [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92096
     - [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92097
- [ ] creating the administrative interface
     - [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92098
- [ ] Add maint windows
     - [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92099
     - [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92100
- [ ] Notify
     - [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/92101

**Comment by mtcA6** [comment]
Tom is going to break this down into its component parts

**Comment by mtcA6** [comment]
@tpharrison I modified the AC on this ticket so it was about breaking down this ticket into it's component parts.

I've added the tickets we made together to two different epics, either for Code Coverage or for Automation. 

My thinking is, our maintenance period work is really 3 different things:
- Phase1 - 2023 COLA Urgent Updates in#68877
- Phase 2 - Code Coverage in#89678
- Phase 3 - Automation in#92169

I felt this would help us keep the efforts clear and the Code Coverage must happen before we can fully automate e2e AND is more important than the automation so we're ready for any outages:

Here's the automation epic:
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/92169

I think we should close this story out and pull in whatever tickets you end up having time for related to the automation (if IES doesn't end up taking all of your time).
